resources/definitions/zonestar_Z9.def.json

```diff
@@ -0,0 +1,48 @@
+{
+    "name": "ZONESTAR Z9",
+    "version": 2,
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "ZONESTAR",
+        "manufacturer": "ZONESTAR",
+        "file_formats": "text/x-gcode",
+        "platform_offset": [ 0, 0, 0],        
+        "machine_extruder_trains":
+        {
+            "0": "zonestar_mixvtool_0",
+            "1": "zonestar_mixvtool_1",
+            "2": "zonestar_mixvtool_2",
+            "3": "zonestar_mixvtool_3",
+            "4": "zonestar_mixvtool_4",
+            "5": "zonestar_mixvtool_5",
+            "6": "zonestar_mixvtool_6",
+            "7": "zonestar_mixvtool_7"

```

Comment:
According to [this review](https://zeroair.org/2019/01/09/zonestar-z8xm2-multi-material-3d-printer-kit-review/), the printer has a maximum of 2 print heads. Is there a printer with 8 print heads, or did you make a mistake here?

Reply:
we choose to use VTOOLs to replace of the "real extruders", the customer will be easy  to do mixing color print with the Virtual Extruder. Please keep these settings if possible, thanks.

Reply:
What do you mean with "VTOOLs"? That is not a term in Cura's architecture.

If the user wants to change the colour of their materials, they are expected to change the material profile, rather than the extruder an object is printed with. This works better with Cura's workflow and profile system. Right now, a custom profile that the user creates would save settings for each extruder separately and won't be available if any of the extruders has a different material type in.